examples_ch59x/debugprintfdemo/funconfig.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _FUNCONFIG_H
+#define _FUNCONFIG_H
+
+#define FUNCONF_USE_HSI         0 // CH592 does not have HSI
+#define FUNCONF_USE_HSE         1
+#define FUNCONF_DEBUG_HARDFAULT 0
```

Comment:
Why not?  (Just wondering)?

Reply:
That's based solely on a quick search in the datasheet. There is HSE, LSE and LSI there, but no HSI. It might be just undocumented, but I'm in no way an expert on these things.

Reply:
Also the CLKType enum does not mention an HSI, so if it's there it's well hidden..